packages/xo-server-test/src/vm/index.spec.js

```diff
@@ -1,33 +1,32 @@
 /* eslint-env jest */
 
-import { mapValues } from 'lodash'
-
 import config from '../_config'
 import xo from '../_xoConnection'
 
 describe('vm', () => {
   describe('.set() :', () => {
-    const PROPERTIES_TO_SET_TO_NULL = {
-      coresPerSocket: 1,
-      cpuCap: 1,
-    }
     it('set properties to null', async () => {
       const id = await xo.createTempVm({
+        coresPerSocket: 1,
+        cpuCap: 1,
         name_label: 'XO Test',
-        template: config.templates.default,
-        ...PROPERTIES_TO_SET_TO_NULL,
+        template: config.templates.templateWithoutDisks,
+      })
+
+      expect(xo.objects.all[id]).toMatchObject({
```

Comment:
What makes you think the object is already in the cache?

Reply:
`createTempVm` implementation.

Reply:
It should return the object then.

Reply:
Do you want me to do it in this PR? Because it'll result to unrelated changes.

Reply:
Let's do it first in another PR and then rebase this one on top of it.